### DISCUSSION TITLE
Sorting game lists by number of players (popularity)

### DISCUSSION THREAD

**Discussion by jumphil** [RetroAchievements/RAWeb#1324] [Ideas]
The game list per system should have an additional sorting column for number of total players for each game. This number already exists, it shows next to achievements. This would be very useful for finding all the popular games together (or obscure games if you want).

**Comment by EmoonX** [comment]
This is in my to-do list. Possibly we should be able to add even more info (game rating, # of masters, etc).

**Comment by Jamiras** [comment]
Available with #2908.
